### ISSUE TITLE
Images stretch when side panels closed

### ISSUE THREAD

**Issue by BricoleurSoul** [Seraaron/agora-obsidian-theme#1]
When images are part of a note they are stretched out of shape if the left and/or right panels are closed.
Expected behaviour: image keeps proper proportion irregardless of width of pane.

Examples show in Agora with 
1 with right panel;  (stretching already apparent with just right panel open
2 without panels (severe stretching of images)
and 
3) another theme that shows proper image even when left and right panels are closed.

Otherwise this could become my go to theme!
![Screenshot of Obsidian (2021-10-15, 10-20-05 PM)](https://user-images.githubusercontent.com/43626234/137575028-ac929c7b-818d-42c4-b792-0c3a31ad3c0e.png)
<img width="1309" alt="Screenshot of Obsidian (2021-10-15, 10-20-35 PM)" src="https://user-images.githubusercontent.com/43626234/137575031-6dae4ce6-2677-4a79-a7c4-1d5f30465ccf.png">
<img width="1317" alt="Screenshot of Obsidian (2021-10-15, 10-21-38 PM)" src="https://user-images.githubusercontent.com/43626234/137575035-a74086f7-5e2e-45c2-9b9d-e8b58a283a51.png">

**Comment by Seraaron** [comment]
Hi @BricoleurSoul thanks for the feedback. Unfortunately, I've not been able to re-produce this behavior. Can you tell me a little more about your obsidian settings, or the source of the image? Is this a web-hosted image or local, for example?

Alternatively, try searching for the `/* SNIPPET: Enlarge image on hover by @den */` section of the code in your obsidian vault .obsidian/themes/Agora.css file and either try playing with some of the numbers there, or delete that section of the code entirely and see if that fixes the issue?

**Comment by BricoleurSoul** [comment]
This is showing up in pages generated by the Official Readwise Plugin.
the image is remote - this is the code:

![rw-book-cover] https://images-na.ssl-images-amazon.com/images/I/512IyJYXcVL._SL200_.jpg


 ![rw-book-cover](https://images-na.ssl-images-amazon.com/images/I/512IyJYXcVL._SL200_.jpg)

I have tested on 14 themes from minimal to very complex  (including default obsidian) and none of them exhibit the stretching  present in display that happens with Agora.

I did comment out the /* SNIPPET: Enlarge image on hover by @den */ section of the code  and that did indeed resolve the issue on Agora.

**UPDATE**
I restored the image hover code and looked at the handling of other remote images in some of my notes (remote images not hosted on amazon.com) and they present correctly!  Just the type of quirky behaviour designed to drive one crazy....

**Comment by Seraaron** [comment]
Yeah, I can't seem to reproduce the issue sorry, even when I try with your own image link there. So I'm really not sure how to help, sorry!

All I can offer is that you maybe make sure you're using the most up to date version of obsidian, open a clean vault, copy across the culprit markdown file into that new vault, and try a fresh install of Agora and see if there's still an issue? Maybe one of your other plugins or settings is interfering in some way...?

**Comment by BricoleurSoul** [comment]
Yes completely up to date on Obsidian.
As I noted the issue doesn't arrive:
a) in 14 other Themes that I have selected
b) when I remove the hover code css code for your plug-in in your theme either
c) in any other notes with images other than those pages generated by use of the readies-offical plugin
....
so after digging into the readwise plug in I found this snippet in their css:

img[alt=rw-book-cover] {
    **max-height: 200px;**
}

I would say this is the root - your hover css tries to do its job but would seem to be overruled by this max-height constraint.

Sorry for wild goose chase I sent you on but now in a quandary - I can't work without Readwise but I really do like your Theme -I wonder if I can guess which is probably to get updated the least and manually change the css each time it is updated...

Or is there a hack to get around this (sorry not a programmer so have no idea about code let alone css...)

At any rate, thanks for your time on this - I will now close the issue.
